OracleWebLogic/dockerfiles/12.2.1.3/README.md

```diff
@@ -1,105 +1,86 @@
 
-  2. Verify that you now have this image in place with:
+	username=myudminsername
+	password=myadminpassword
 
-        `$ docker images`
+**Note**: Oracle recommends that the domain.properties file be deleted or secured after the container and the WebLogic server are started so the username and password are not inadvertently exposed.
 
-  3. Start a container from the image created in step 1.
-     You can override the default values of the following parameters during runtime with the `-e` option:
-      * `ADMIN_NAME`     (default: `AdminServer`)
-      * `ADMIN_PORT`     (default: `7001`)
-      * `ADMIN_USERNAME` (default: `weblogic`)
-      * `ADMIN_PASSWORD` (default: Auto Generated)
-      * `DOMAIN_NAME`    (default: `base_domain`)
-      * `DOMAIN_HOME`    (default: `/u01/oracle/user_projects/domains/base_domain`)
+#### Start the Container
+Start a container from the image created in step 1.
+You can override the default values of the following parameters during runtime with the `-e` option:
 
-**NOTE**: To set the `DOMAIN_NAME`, you must set both `DOMAIN_NAME` and `DOMAIN_HOME`.
+      * `ADMIN_NAME`                  (default: `AdminServer`)
+      * `ADMIN_LISTEN_PORT`           (default: `7001`)
+      * `DOMAIN_NAME`                 (default: `base_domain`)
+      * `DOMAIN_HOME`                 (default: `/u01/oracle/user_projects/domains/base_domain`)
+      * `ADMINISTRATION_PORT_ENABLED` (default: `true`)
+      * `ADMINISTRATION_PORT`         (default: `9002`)
 
-        $ docker run -d -p 7001:7001 -e ADMIN_USERNAME=weblogic -e ADMIN_PASSWORD=welcome1 -e DOMAIN_HOME=/u01/oracle/user_projects/domains/abc_domain -e DOMAIN_NAME=abc_domain oracle/weblogic:12.2.1.3-developer
+**NOTE**: To set the `DOMAIN_NAME`, you must set both `DOMAIN_NAME` and `DOMAIN_HOME`. For security the Administration port 9002 is enabled by default, if you would like to disable it set 'ADMINISTRTATION_PORT_ENABLED' to false. If you intend to run these images in production, you must change the Production Mode to `production`.
 
-  4. Run the Administration Console:
+	$docker run -d -p 7001:7001 -p 9002:9002  -v `HOST PATH where the domain.properties file is`:/u01/oracle/properties -e ADMINISTRATION_PORT_ENABLED=true -e DOMAIN_HOME=/u01/oracle/user_projects/domains/abc_domain -e DOMAIN_NAME=abc_domain oracle/weblogic:12.2.1.3-developer
 
-        `$ docker inspect --format '{{.NetworkSettings.IPAddress}}' <container-name>`
+Run the Administration Console:
 
-        This returns the IP address of the container (for example, `xxx.xx.x.x`). Go to your browser and enter `http://xxx.xx.x.x:7001/console`
+        $ docker inspect --format '{{.NetworkSettings.IPAddress}}' <container-name>
 
+Go to your browser and enter `https://xxx.xx.x.x:9002/console` your browser will request for you to accept Security Exception. To avoid the Security Exception you must update the WebLogic server SSL configuration with a custom identity certificate.
```

Comment:
...enter `https://xxx.xx.x.x:9002/console`. Your browser will request for you to accept the Security Exception.